core/ui/TestCases/DefaultTemplate.tid

```diff
@@ -72,7 +72,7 @@ code-body: yes
 		<%endif%>
 		<div class="tc-test-case-panes">
 			<div class="tc-test-case-source">
-				<$macrocall $name="tabs" tabsList="[all[tiddlers]sort[]] -[prefix[$:/state/]] -Description -Narrative -Output Output +[putfirst[]] -[has[plugin-type]]" state=<<state>> default="Output" template="$:/core/ui/testcases/DefaultTemplate/SourceTabs"/>
+				<$macrocall $name="tabs" tabsList="[all[tiddlers]sort[]] -[prefix<state>] -[prefix[$:/state/popup/export]] -Description -Narrative -Output Output +[putfirst[]] -[has[plugin-type]]" state=<<state>> default="Output" template="$:/core/ui/testcases/DefaultTemplate/SourceTabs"/>
```

Comment:
Since we are at it, I think we should also move the _strange_ construction  `-Output Output +[putfirst[]]` to the "front" of the filter string like so: 

`[all[tiddlers]sort[]] Output +[putfirst[]]` which allows us to remove the `-Output` from the filter string. 

@Jermolene -- what do you think?

Reply:
Yes that is clearer